### ISSUE TITLE
Add an "Skip this version" option to update dialog.

### ISSUE THREAD

**Issue by ontherunvaro** [Thomas101/wmail#637]
The update dialog is annoying because it keeps showing up if you don't update. I know there's an "after restart" option but that is still not enough.

A solution is adding a "Skip this version" option that would not remind you again for the same version. Many other applications already do this.

* Which version of WMail are you using? 2.3.0
* Which Operating System are you using? Ubuntu 14.04

P.D.: Thank you for your amazing app!

**Comment by Thomas101** [comment]
Good idea! Also if you want to disable it completely you can do this in settings!
